karate-demo/src/test/java/demo/DemoTestParallel.java

```diff
@@ -19,29 +19,30 @@
  */
 // important: do not use @RunWith(Karate.class) !
 public class DemoTestParallel {
-    
+
     @BeforeClass
-    public static void beforeClass() throws Exception {        
+    public static void beforeClass() throws Exception {
         TestBase.beforeClass();
-    } 
-    
+    }
+
     @Test
     public void testParallel() {
         Results results = Runner.path("classpath:demo")
                 .outputCucumberJson(true)
                 .karateEnv("demo")
-                .tags("~@ignore").parallel(5);
+                .tags("~@ignore")
+                .parallel(5);
         generateReport(results.getReportDir());
-        assertTrue(results.getErrorMessages(), results.getFailCount() == 0);        
+        assertTrue(results.getErrorMessages(), results.getFailCount() == 0);
     }
-    
-    public static void generateReport(String karateOutputPath) {        
+
+    public static void generateReport(String karateOutputPath) {
         Collection<File> jsonFiles = FileUtils.listFiles(new File(karateOutputPath), new String[] {"json"}, true);
-        List<String> jsonPaths = new ArrayList(jsonFiles.size());
+        List<String> jsonPaths = new ArrayList<String>(jsonFiles.size());
```

Comment:
the `String` is not required, just the `<>` should do.

Reply:
Thanks for your feedback!
I do apologize for that, next time, I hope, I will do more valuable than this one.